README.md

```diff
@@ -246,6 +242,9 @@ All listed key-value pairs are necessary. You can keep
 some of them in `.pcconfig` in your home directory,
 and others in `pcconfig.json` in your project.
 
+To find `pcconfig.json`, `pcsync` needs to know your target directory,
+so `PLAYCANVAS_TARGET_DIR` can only be set in `.pcconfig` or an environment variable.
+
```

Comment:
I would phase this:
```
All listed key-value pairs are necessary. 

`PLAYCANVAS_TARGET_DIR` must be in `.pcconfig` in the home directory or an environment variable.

The reminder can be in `pcconfig.json` in your project target directory or all key-value pairs can be in `.pcconfig` in the home directory.
```

Reply:
I've merged what you suggest with what I had. Please feel free to suggest more readme edits if necessary, probably in a separate pr.